### PR TITLE
Refresh Pancake Sort

### PR DESCRIPTION
**Pull request by srb1986**
Refreshed the "Pancake Sorter" Implementation in line with other "sorter" refreshes.

One File Deletion
--> pancake_sort.cs
Two File Additions
--> Algorithms.Tests/Sorters/PancakeSorterTests.cs
--> Algorithms/Sorters/PancakeSorter.cs

Tested with Random Integer Array

Built successfully in Debug and Release

### CODE REVIEW

File: Algorithms/Sorters/PancakeSorter.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            }
+        }
+
+        // Returns index of the maximum element 
+        // in array[0..n-1]
+        //
+        private int FindMax(T[] array, int n, IComparer<T> comparer)
+        {
+            int mi, i;
+            for (mi = 0, i = 0; i < n; ++i)
```
Comment (siriak) [inline Algorithms/Sorters/PancakeSorter.cs]:
```suggestion
            for (var i = 0; i < n; i++)
```

File: Algorithms/Sorters/PancakeSorter.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                i--;
+            }
+        }
+
+        // Returns index of the maximum element 
+        // in array[0..n-1]
+        //
+        private int FindMax(T[] array, int n, IComparer<T> comparer)
+        {
+            int mi, i;
```
Comment (siriak) [inline Algorithms/Sorters/PancakeSorter.cs]:
```suggestion
            var mi = 0;
```

**Comment by srb1986** [comment]
I will modify the PancakeSorter.cs file in line with codacy-bot's recommendation:

Codacy Issue found: Do not update the loop counter 'mi' within the loop body.

**Comment by srb1986** [comment]
Reopened Pull Request

**Pull request merged**